keyboards/mawaeg/sharkropad/sharkropad.c

```diff
@@ -0,0 +1,125 @@
+/* Copyright 2025 @mawaeg
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+#include <ctype.h>
+#include <lib/lib8tion/lib8tion.h>
+#include <rgb_matrix.h>
+
+#if defined(RGB_MATRIX_EFFECT)
```

Comment:
If you're up for targeting the `develop` branch instead, you can remove lines 22-77 and use #25344 instead to get the effect names.